src/ssd1306-element.stories.ts

```diff
@@ -0,0 +1,18 @@
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/web-components';
+import { html } from 'lit-html';
+import './ssd1306-element';
+
+storiesOf('Ssd1306 Element', module)
```

Comment:
We can remove "Element" from the name (just to stay consistent with the other stories), also call it SSD1306 (like we used uppercase in LED and LCD1602)